include/SDL3/SDL_system.h

```diff
@@ -90,10 +90,6 @@ typedef SDL_bool (SDLCALL *SDL_WindowsMessageHook)(void *userdata, MSG *msg);
  */
 extern SDL_DECLSPEC void SDLCALL SDL_SetWindowsMessageHook(SDL_WindowsMessageHook callback, void *userdata);
 
-#endif /* defined(SDL_PLATFORM_WIN32) || defined(SDL_PLATFORM_GDK) */
-
-#if defined(SDL_PLATFORM_WIN32) || defined(SDL_PLATFORM_WINGDK)
```

Comment:
This isn't correct. SDL_PLATFORM_WINGDK is distinct from the Xbox platforms, which are covered in SDL_PLATFORM_WINDOWS, so you still need this.

Reply:
It's a subtle distinction between SDL_PLATFORM_GDK, which covers all GDK related platforms, and SDL_PLATFORM_WINGDK, which is just GDK on desktop.

Reply:
You'll need to go back through and undo all your `defined(SDL_PLATFORM_WIN32) || defined(SDL_PLATFORM_WINGDK)` changes.

Reply:
Shoot, I knew I was going to catch a few of these by accident. I tried to only get the ones that were `WIN32 || GDK` and not the `WINGDK` ones. Human error, I'll double-check the patch.

Reply:
Fixed now, I think (I moved the `#include SDL_windows.h` lines back to the WINGDK check, too, but these were probably safe to leave alone.)